dcos_test_utils/dcos_cli.py

```diff
@@ -92,6 +94,45 @@ def clear_cli_dir():
         if os.path.exists(path):
             shutil.rmtree(path)
 
+    def exec(self, cmd: str, stdin=None, check=True) -> subprocess.CompletedProcess:
+        """Execute CLI command and processes result.
+
+        This method expects that process won't block.
+
+        :param cmd: Program and arguments
+        :type cmd: str
+        :param stdin: File to use for stdin
+        :param check: Does it check for raised errors
+        :type stdin: File
+        :returns: A tuple with stdout and stderr
+        :rtype: subprocess.CompletedProcess
```

Comment:
What about introducing the exception that may be raised?

Reply:
You mean on the docs, or code change? Can you exemplify a bit?

Reply:
I mean `exec` may raise an exception, can you add the description of the exception in docstring